src/config.ts

```diff
@@ -41,10 +44,34 @@ if (
     );
 }
 
+if (!INVENTORY_TEZOS_SECRET_KEY) {
+    throw new Error(
+        "Please provide an uncrypted private key to sign transactions with via INVENTORY_SECRET_KEY."
+    );
+}
+
+if (!INVENTORY_TEZOS_PUBLIC_KEY_HASH) {
+    throw new Error(
+        "Please provide a public key hash for the contract owner via INVENTORY_TEZOS_PUBLIC_KEY_HASH."
+    );
+}
+
+if (
+    INVENTORY_CONTRACT_ADDRESS &&
+    !validateAddress(INVENTORY_CONTRACT_ADDRESS)
+) {
+    throw new Error(
+        "Please provide a valid KT1 address to access the INVENTORY via INVENTORY_CONTRACT_ADDRESS or leave blank for the contract to be automatically deployed"
```

Comment:
so here, we won't be deploying an inventory contract upon startup, contrary to warehouse.
there is only one warehouse contract for the whole platform, but there is at least one inventory per user that we create.
So the inventory contract isn't deployed when the application starts, only when the related handler is called